lib/rules/jsx-curly-spacing.js

```diff
@@ -10,13 +10,24 @@
 
 module.exports = function(context) {
   var spaced = context.options[0] === 'always';
+  var multiline = context.options[1] ? context.options[1].allowMultiline : true;
```

Comment:
This is a little fragile and relies on the fact that it's the only option and that no one would pass `{}` as the second parameter, I can change it to leverage `hasOwnProperty` if you want.